.github/workflows/terraform-observe_scheduler.yaml

```diff
@@ -27,33 +32,55 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Build matrix
-        id: matrix
-        shell: bash
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: download python script
+        id: matrix_v2
         run: |
-          DIRS="$(find . -path "*tftests/*" ! -path "*.terraform*" -type d )"
-          [[ ${#DIRS} > 0 ]] && echo "directories=[\"${DIRS//$'\n'/\",\"}\"]" >> "$GITHUB_OUTPUT" || echo "directories=[]" >> "$GITHUB_OUTPUT"
+          curl -O https://raw.githubusercontent.com/observeinc/.github/${{ inputs.script_branch }}/.github/scripts/main.py
```

Comment:
Hmm would it make more sense to use `@` tags to call this workflow specific branch from other repos?
Eg: If user calls this workflow, specifies a different branch name with `@` but forgets to specify the python branch, it would be assumed the branch that script runs is the user's branch, and not main as defaulted here. 



See: https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow